exercise_08/formatStringByWordsLength.js

```diff
@@ -16,3 +16,23 @@
  * CASE #2 * input: "This is short one."
  * output: "this is short one."
  */
+
+/**
+ * #8 :: Formats a text string according to the number of words.
+ * If there are more than 5 words, return the string in uppercase.
+ * If there are 5 or fewer words, return the string in lowercase.
+ *
+ * @param {string} sentence - The text string to format.
+ * @returns {string} - The string formatted according to the number of words.
+ */
+const formatStringByWordsLength = function (sentence) {
+    const words = sentence.split(/\s+/);
+    if (words.length > 5) {
```

Comment:
Avoid magic numbers. Use descriptive constants
```suggestion
    if (words.length > THRESHOLD) {
```